commit cda700b4c8a8a60362f14b79243eaaadbebd09ef
Author: Mike Frysinger <vapier@gentoo.org>
Date:   Tue Feb 24 00:10:42 2015 -0500

    hppa: fix build failure with RTLD_PRIVATE_ERRNO
    
    Pull in dl-sysdep.h like every other linux sysdep.h header does when it
    wants to use RTLD_PRIVATE_ERRNO.

--- a/sysdeps/unix/sysv/linux/hppa/sysdep.h
+++ b/sysdeps/unix/sysv/linux/hppa/sysdep.h
@@ -21,6 +21,9 @@
 #include <asm/unistd.h>
 #include <sysdeps/generic/sysdep.h>
 
+/* Defines RTLD_PRIVATE_ERRNO.  */
+#include <dl-sysdep.h>
+
 /* In order to get __set_errno() definition in INLINE_SYSCALL.  */
 #ifndef __ASSEMBLER__
 #include <errno.h>
